doc/charm++/manual.rst

```diff
@@ -11646,6 +11723,20 @@ directory. Pathname resolution is performed as follows:
 #. The system tries to locate this program (with modified pathname and
    appended extension if specified) on all nodes.
 
+
+Instructions to run Charm++ programs for Specific Builds
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+UCX
+^^^
+When Charm++ is built with UCX using Simple PMI, Slurm PMI or Slurm PMI2, ensure that
+an MPI implementation which is not OpenMPI is loaded in the environment. Use the mpiexec/mpirun
+launcher provided by the non-OpenMPI implementation to launch programs. Alternatively, you
+can also use any system provided launchers.
```

Comment:
maybe add here:  'Note that due to bug #2477, binaries built with UCX cannot be run without mpiexec/mpirun.'

Reply:
Done.